actionpack/lib/abstract_controller/helpers.rb

```diff
@@ -54,6 +54,10 @@ def inherited(klass)
       # In a view:
       #  <% if logged_in? -%>Welcome, <%= current_user.name %><% end -%>
       #
+      # Note that helpers defined with this method are ran outside of the
```

Comment:
The problem on that issue is not the use of `helper_method` but the usage of `helpers` inside the helper method. I think we should document on `helpers`.